rero_ils/modules/acq_accounts/jsonschemas/acq_accounts/acq_account-v0.0.1.json

```
@@ -31,19 +34,75 @@
     },
     "name": {
       "title": "Name",
-      "description": "Name of the acquisition account.",
+      "description": "Common name of the acquisition account.",
       "type": "string",
       "minLength": 1,
       "form": {
         "focus": true
       }
     },
-    "description": {
-      "title": "Description",
-      "description": "Description of the acquisition account.",
+    "number": {
+      "title": "Account number",
+      "description": "Reference account number in accounting system.",
       "type": "string",
       "minLength": 1
     },
+    "allocated_amount": {
+      "title": "Allocated amount",
+      "description": "The amount allocated for the acquisition account.",
+      "type": "number",
+      "minimum": 0
+    },
+    "encumbrance_exceedance": {
+      "title": "Emcumbrance exceedance",
+      "description": "The pourcentage of account amount than linked pending orders can exceed.",
+      "type": "number",
+      "minimum": 0,
+      "maximum": 100,
+      "default": 0,
+      "form": {
+        "templateOptions": {
+          "addonRight": {
+            "text": "%"
+          }
+        }
+      }
+    },
+    "expenditure_exceedance": {
+      "title": "Expenditure exceedance",
+      "description": "The pourcentage of account amount than linked paid invoices can exceed.",
+      "type": "number",
+      "minimum": 0,
+      "maximum": 100,
+      "default": 0,
+      "form": {
+        "templateOptions": {
+          "addonRight": {
+            "text": "%"
+          }
+        }
+      }
+    },
+    "parent": {
+      "title": "Parent account",
+      "type": "object",
+      "properties": {
+        "$ref": {
+          "title": "Account URI",
+          "type": "string",
+          "pattern": "^https://ils.rero.ch/api/acq_accounts/.*?$"
+        }
+      },
+      "form": {
+        "remoteOptions": {
+          "type": "acq_accounts"
+        },
+        "placeholder": "Choose a parent account",
```

```suggestion
        "placeholder": "Choose a parent account…",
```
To suggest an action? I don't know...